fuel_test/prep_env.sh

```diff
@@ -7,16 +7,26 @@ set -eu
 [ $DEBUG == "on" ] && set -x
 
 function restore_fm {
-	#Restore snapshot that has test machine's ssh public key
+	# Restore fuel master xva
 	local xs_host="$1"
 	local fm_name="$2"
-	local fm_snapshot="$3"
+	local fm_mnt="$3"
+	local fm_xva="$4"
 	ssh -qo StrictHostKeyChecking=no root@$xs_host \
 	'
 	set -eux
 	vm_uuid=$(xe vm-list name-label="'$fm_name'" --minimal)
-	snapshot_uuid=$(xe snapshot-list name-label="'$fm_snapshot'" snapshot-of="$vm_uuid" --minimal)
```

Comment:
Why?  Reverting to a snapshot is very quick; importing an XVA is much much slower.